@uportal/eyebrow-user-info/babel.config.js

```diff
@@ -1,3 +1,7 @@
-module.exports = {
-  presets: ["@vue/app"]
+module.exports = function(api) {
+  api.cache.never();
+  return {
+    plugins: ["babel-plugin-transform-custom-element-classes"],
+    presets: ["@babel/preset-env"]
```

Comment:
`@vue/app` preset may work better here and avoid needing `babel-polyfill`.
See https://github.com/uPortal-contrib/uPortal-web-components/pull/109/commits/37b0f70635a132b0d2af9453f1b132b40453cc5d